radarr/hatch.toml

```
@@ -0,0 +1,7 @@
+[envs.default]
+e2e-env = false
+
+[env.collectors.datadog-checks]
+
+[[envs.default.matrix]]
+python = ["3.8"]
```

The Agent's Python has been upgraded to 3.9, that's why the tests are failing now. The fix is simple:

```suggestion
python = ["3.9"]
```